cloud_governance/policy/helpers/aws/aws_policy_operations.py

```diff
@@ -27,7 +28,7 @@ def __init__(self):
         self._ec2_operations = EC2Operations(region=self._region)
         self._cloudwatch = CloudWatchOperations(region=self._region)
         self._resource_pricing = ResourcesPricing()
-        self.cost_savings_tag = [{'Key': 'cost-savings', 'Value': 'true'}]
+        self.cost_savings_tag = [{'Key': 'cost-savings', 'Value': self.policy_name}]
```

Comment:
Do you think it better to create another tag ?
policy:policy_name

Reply:
No need.
 cost-savings: policy_name specifies the savings amount on that policy.